File: SDKs/plaid.com/2020-09-14_1.20.6/python/src/sdk/models/operations/processorbanktransfercreate.py
import dataclasses
from typing import Any,Optional
from ..shared import processorbanktransfercreaterequest as shared_processorbanktransfercreaterequest


@dataclasses.dataclass
class ProcessorBankTransferCreateRequest:
    request: shared_processorbanktransfercreaterequest.ProcessorBankTransferCreateRequest = dataclasses.field(metadata={'request': { 'media_type': 'application/json' }})
    

@dataclasses.dataclass
class ProcessorBankTransferCreateResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    error: Optional[dict[str, Any]] = dataclasses.field(default=None)
    processor_bank_transfer_create_response: Optional[dict[str, Any]] = dataclasses.field(default=None)
    

File: SDKs/plaid.com/2020-09-14_1.20.6/typescript/src/sdk/models/operations/processorbanktransfercreate.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class ProcessorBankTransferCreateRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request: shared.ProcessorBankTransferCreateRequest;
}


export class ProcessorBankTransferCreateResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  error?: Record<string, any>;

  @SpeakeasyMetadata()
  processorBankTransferCreateResponse?: Record<string, any>;

  @SpeakeasyMetadata()
  statusCode: number;
}
